Tyro/JSONKeypath.swift

```diff
@@ -51,7 +51,7 @@ extension JSONKeypath : CustomStringConvertible {
 }
 
 extension JSONKeypath {
-    func resolve(dictionary : Dictionary<String, JSONValue>) -> JSONValue? {
+    public func resolve(dictionary : Dictionary<String, JSONValue>) -> JSONValue? {
```

Comment:
Definitely. I thought it would be good to made public so that it can be used in outside implementations if necessary.